### PR TITLE
Knowledgebase index and view now displays, added some SQL data

### PR DESCRIPTION
**Pull request by patch7331**
It works!

### CODE REVIEW

File: WEB-INF/classes/uon/seng2050/assignment/model/Article.java [hunk 1]
```diff
@@ -10,7 +11,8 @@
   private String body;
   private String answer;
   private int helpfulness;
-
+  private Date published;
```
Comment (sean0x42) [inline WEB-INF/classes/uon/seng2050/assignment/model/Article.java]: I've moved dates to the Model class, so there is no need to track these here.

File: WEB-INF/sql/create_tables.sql [hunk 2]
```diff
@@ -22,4 +22,15 @@ CREATE TABLE Issues (
...
 );
 
+CREATE TABLE Articles (
+  id          CHAR(36)     PRIMARY KEY,
+  title       VARCHAR(128) NOT NULL,
+  body        TEXT         NOT NULL,
+  answer      TEXT         NOT NULL,
+  helpfulness INT          NOT NULL DEFAULT 0,
+  published   DATE         NOT NULL,
+  updated     DATE         NOT NULL
```
Comment (sean0x42) [inline WEB-INF/sql/create_tables.sql]: For right now, no dates. I'll add them later once I make models automatically update these values.

**Review by sean0x42 — CHANGES_REQUESTED** [review]
Just some minor changes please

File: WEB-INF/classes/uon/seng2050/assignment/model/Article.java [hunk 3]
```diff
@@ -11,7 +12,6 @@
   private String answer;
```
Comment (sean0x42) [inline WEB-INF/classes/uon/seng2050/assignment/model/Article.java:14]: This should be a foreign key (UUID). Also just add category and subcategory please :)

**Review by sean0x42 — CHANGES_REQUESTED** [review]

**Comment by patch7331** [comment]
sean pls

**Comment by sean0x42** [comment]
Looks good :tada: :taco: :tajikistan: :tanabata_tree:

**Pull request merged**